docs/src/dataDisplay/telephoneLink/examples/exampleTelephoneLink.jsx

```diff
@@ -0,0 +1,17 @@
+import {
+    TelephoneLink,
+} from 'react-cm-ui';
+import React from 'react';
+
+function ExampleTelephoneLink() {
+    return (
+        <TelephoneLink
+            className="block_name--element_name"
+            formattedNumberSample="MX"
```

Comment:
[This issue}(https://github.com/saddlebackdev/react-cm-ui/pull/275#discussion_r635203008) still persists.
Should be something like:
```JavaScript
formattedNumber="(949) 609-8002"
number="+19496098002"
```

Reply:
Fixed.

Reply:
Looks good.  Resolving.